<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View model</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menu_Bounds">
     <property name="title">
      <string>Bounds</string>
     </property>
     <addaction name="action_BoundingBox"/>
     <addaction name="action_Bounds"/>
    </widget>
    <widget class="QMenu" name="menuScalars">
     <property name="title">
      <string>Scalars</string>
     </property>
     <addaction name="action_log"/>
     <addaction name="action_normal"/>
    </widget>
    <widget class="QMenu" name="menu_Display">
     <property name="title">
      <string>Display Format</string>
     </property>
     <addaction name="action_PyVista"/>
     <addaction name="action_UBC"/>
    </widget>
    <addaction name="menu_Display"/>
    <addaction name="separator"/>
    <addaction name="action_Wireframe"/>
    <addaction name="menu_Bounds"/>
    <addaction name="action_Orientation_Marker"/>
    <addaction name="menuScalars"/>
    <addaction name="separator"/>
    <addaction name="action_Camera"/>
    <addaction name="separator"/>
    <addaction name="action_Clear"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_Threshold"/>
    <addaction name="action_Crop"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="action_AddPoints"/>
    <addaction name="action_AddLines"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuEditor"/>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_XOZview"/>
   <addaction name="action_XOYview"/>
   <addaction name="action_YOZview"/>
   <addaction name="action_Isometric"/>
   <addaction name="separator"/>
   <addaction name="action_Bounds"/>
   <addaction name="separator"/>
   <addaction name="action_PyVista"/>
   <addaction name="action_UBC"/>
   <addaction name="separator"/>
   <addaction name="action_log"/>
   <addaction name="action_normal"/>
   <addaction name="separator"/>
   <addaction name="action_Threshold"/>
   <addaction name="action_Crop"/>
  </widget>
  <action name="action_Clear">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="action_Camera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="action_XOZview">
   <property name="text">
    <string>XOZ view</string>
   </property>
  </action>
  <action name="action_XOYview">
   <property name="text">
    <string>XOY view</string>
   </property>
   <property name="toolTip">
    <string>XOY view</string>
   </property>
  </action>
  <action name="action_YOZview">
   <property name="text">
    <string>YOZ view</string>
   </property>
   <property name="toolTip">
    <string>YOZ view</string>
   </property>
  </action>
  <action name="action_Isometric">
   <property name="text">
    <string>Isometric</string>
   </property>
   <property name="toolTip">
    <string>Isometric</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_Threshold">
   <property name="text">
    <string>Threshold</string>
   </property>
  </action>
  <action name="action_Wireframe">
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="toolTip">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="action_BoundingBox">
   <property name="text">
    <string>Bounding box</string>
   </property>
  </action>
  <action name="action_Bounds">
   <property name="text">
    <string>Bounds</string>
   </property>
  </action>
  <action name="action_Orientation_Marker">
   <property name="text">
    <string>Orientation Marker</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="action_Crop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="action_log">
   <property name="text">
    <string>log</string>
   </property>
  </action>
  <action name="action_normal">
   <property name="text">
    <string>normal</string>
   </property>
  </action>
  <action name="action_AddPoints">
   <property name="text">
    <string>Points</string>
   </property>
  </action>
  <action name="action_PyVista">
   <property name="text">
    <string>PyVista</string>
   </property>
  </action>
  <action name="action_UBC">
   <property name="text">
    <string>UBC</string>
   </property>
  </action>
  <action name="action_AddLines">
   <property name="text">
    <string>Lines</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
